examples/.gitignore

```diff
@@ -3,6 +3,7 @@ libhost.a
 libapp.so
 dynhost
 preprocessedhost
```

Comment:
We shouldn't need `preprocessedhost` anymore either!
```suggestion
```

Reply:
I'll take care of that in my next PR.